plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/Library.java

```diff
@@ -1,21 +1,33 @@
 package com.google.cloud.tools.eclipse.appengine.libraries;
 
+import java.net.URI;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
 import org.eclipse.core.runtime.IPath;
 import org.eclipse.core.runtime.Path;
 
 import com.google.common.base.Preconditions;
 
 /**
- * Represents a library that can be added to App Engine projects. E.g. AppEngine Endpoints library.
+ * Represents a library that can be added to App Engine projects, e.g. AppEngine Endpoints library.
  *
  */
 public class Library {
```

Comment:
final, per Effective Java Item 17: Design and document for inheritance or else prohibit it